rustls/src/crypto/hpke.rs

```diff
@@ -39,31 +39,32 @@ pub struct HpkeSuite {
 
 /// An HPKE instance that can be used for base-mode single-shot encryption and decryption.
 pub trait Hpke: Debug + Send + Sync {
-    /// Seal the provided `plaintext` to the recipient public key `pk_r` with application supplied
+    /// Seal the provided `plaintext` to the recipient public key `pub_key` with application supplied
     /// `info`, and additional data `aad`.
     ///
     /// Returns ciphertext that can be used with [Self::open] by the recipient to recover plaintext
-    /// using the same `info` and `aad` and the private key corresponding to `pk_r`.
+    /// using the same `info` and `aad` and the private key corresponding to `pub_key`. RFC 9180
+    /// refers to `pub_key` as `pk_r`.
```

Comment:
I thought `pk_r` was the Rusty spelling of `pkR` in the RFC? For searchability, I'd try to retain the RFC spelling in doc comments.

Reply:
Oops, you're right. Fixed up both `sk_r` and `pk_r` refs to be `skR` and `pkR` throughout.